src/docs/markdown/caddyfile/directives/log_append.md

```diff
@@ -37,3 +37,21 @@ example.com {
 	}
 }
 ```
+
+Display in the logs, which reverse proxy upstream was effectively used (either `node1:80`, `node2:80` or `node3:80`) and
```

Comment:
The example only uses `host` (not `hostport`) so the port isn't included in the output

```suggestion
Display in the logs, which reverse proxy upstream was effectively used (either `node1`, `node2` or `node3`) and
```